R/html-tweak.R

```diff
@@ -251,38 +251,46 @@ tweak_homepage_html <- function(html, strip_header = FALSE) {
 
 # Mutates `html`, removing the badge container
 badges_extract <- function(html) {
-  # First try specially named div;
-  x <- xml2::xml_find_first(html, "//div[@id='badges']")
+  # First try specially named element;
+  x <- xml2::xml_find_first(html, "//*[@id='badges']")
```

Comment:
I'd prefer to keep this constrained to `<divs>`

Reply:
my use case, as shown before, is a `<ul/>` what’s your rationale to constrain it to an arbitrary tag?

Reply:
Reducing the chance of false positives — you can always deliberately wrap in a div if needed, and I think it's always best to be as restrictive as possible with any selector.

Reply:
Ah sure, this only affects the ID branch, the comment can still wrap anything.

I still think that an ID (which by definition can only appear once on a page) is unique enough, but sure, I’ll revert it.